core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -231,6 +231,9 @@ class KafkaConfig private(doLog: Boolean, val props: util.Map[_, _])
   private val _remoteLogManagerConfig = new RemoteLogManagerConfig(this)
   def remoteLogManagerConfig = _remoteLogManagerConfig
 
+  private val _quorumConfig = new QuorumConfig(this)
+  def quorumConfig = _quorumConfig
```

Comment:
nit: please declare the type explicitly

Reply:
> nit: please declare the type explicitly

Done, I have declared the `QuorumConfig` type, PTAL